### PR TITLE
Update release notes for 23.0.2

### PR DESCRIPTION
**Pull request by mwittgen**

### CODE REVIEW

**Comment by timj** [comment]
It looks like the 23.0.1 on https://pipelines.lsst.io/v/v23_0_2/index.html also needs to be 23.0.2.

**Comment by mwittgen** [comment]
https://jira.lsstcorp.org/browse/DM-34982

File: releases/v23_0_0.rst [hunk 1]
```diff
@@ -13,13 +13,13 @@ Release 23.0.1 (2022-04-06)
...
+| Git tag                                   | 23.0.2     |
 +-------------------------------------------+------------+
-| :doc:`EUPS distrib </install/newinstall>` | v23\_0\_1  |
+| :doc:`EUPS distrib </install/newinstall>` | v23\_0\_2  |
 +-------------------------------------------+------------+
 
 
-This release is based on the ``w_2021_40`` weekly build (September 30, 2021), with 42 tickets backported to 23.0.1 to support the Data Preview 0.2 Data Release (DP0.2).
+This release is based on the ``w_2021_40`` weekly build (September 30, 2021), with 46 tickets backported to 23.0.2 to support the Data Preview 0.2 Data Release (DP0.2).
 For the production of DP0.2, 23.0.0 was used for single-frame processing (the Tasks within the Pipeline subsets ``step1`` and ``step2``), and 23.0.1 was used for coadd-processing (Tasks within the Pipeline subset ``step3``).
```
Comment (yalsayyad) [inline releases/v23_0_0.rst:23]: I suggest:

, and 23.0.1 was used for coadd-processing (Tasks within the Pipeline subset ``step3``), image differencing, and forced photometry (Tasks within the Pipeline subset ``step4``).  23.0.2 was used for associating DIA Objects, their forced photometry,  recalibrating the Source Tables, aggregating global property maps and parquet tables (Tasks within the Pipeline subsets ``step5``,  ``step6``, and ``step7``).

**Review by yalsayyad — APPROVED** [review]
And before you fowardport this to main (yes, I made up a word) let me forwardport the v23.0.1 release notes to main.

**Pull request merged**